payments/stripe/widgets.py

```diff
@@ -8,6 +8,7 @@
 from django.utils.html import format_html
 from django.utils.encoding import force_text
 from django.utils.translation import ugettext_lazy as _
+import django
```

Comment:
Is this import necessary?

Reply:
No, we're not checking django version, so I removed it